spr/spr.go

```diff
@@ -61,20 +61,25 @@ func (sd *stackediff) AmendCommit(ctx context.Context) {
 	if len(localCommits) == 1 {
 		fmt.Fprintf(sd.output, "Commit to amend [%d]: ", 1)
 	} else {
-		fmt.Fprintf(sd.output, "Commit to amend [%d-%d]: ", 1, len(localCommits))
+		fmt.Fprintf(sd.output, "Commit to amend (%d-%d): ", 1, len(localCommits))
 	}
 
 	reader := bufio.NewReader(sd.input)
 	line, _ := reader.ReadString('\n')
 	line = strings.TrimSpace(line)
-	commitIndex, err := strconv.Atoi(line)
-	if err != nil || commitIndex < 1 || commitIndex > len(localCommits) {
-		fmt.Fprint(sd.output, "Invalid input\n")
-		return
+	selectedCommitIndex := 0
+	if len(localCommits) == 1 && line == "" {
+		// do nothing, use the default
```

Comment:
fwiw I don't know how to use go so this was my way of getting something in the right scope.